web/test/test_googlescholar.py

```diff
@@ -19,8 +19,10 @@ class TestGoogleScholar():
     def test_create__success(self):
         link = 'https://scholar.google.ca/citations?user=KffJRdgAAAAJ&hl=en&oi=sra'
         ga = GoogleScholarPageScrape()
-        obj = Faculty(name="Test.Prof", faculty_id=110, email="test@test.com")
+        obj = Faculty(name="Test.Profman", faculty_id=115, email="test@test.com")
```

Comment:
These tests are going to fail as a result of some older changes that I just merged. We will need full_name as another parameter, or we can set full_name to not required in models. Now that I think about it, full name is only used to match nserc grants, so we can set it to not required without issue.